docs/appendix/track.md

```diff
@@ -17,18 +17,20 @@ One of the most-often used traces is to find the origins of health problems like
 Here is the logic for tracking and tracing.
 
 * For an EconomicResource:
-    * after: EconomicEvents affecting it that are inputs to Processes or Transfers
-    * before: EconomicEvents affecting it that are outputs from Processes or Transfers
-* For a Process or Transfer:
+    * after: EconomicEvents affecting it that are inputs to Processes or transfer/move EconomicEvents with the resource defined as the `resourceInventoriedAs`
+    * before: EconomicEvents affecting it that are outputs from Processes or transfer/move EconomicEvents with the resource defined as the `toResourceInventoriedAs`
+* For a Process:
     * after: EconomicEvents that are outputs
     * before: EconomicEvents that are inputs
 * For an EconomicEvent:
     * after: 
-        * a Process or Transfer to which it is an input, or
-        * an EconomicResource which it affected as the output of a Process or Transfer
+        * a Process to which it is an input, or
+        * an EconomicResource which it affected as the output of a Process, or
+        * if it is a transfer or move event, the `toResourceInventoriedAs` EconomicResource
```

Comment:
if it is a transfer or move event, the EconomicResource labelled `toResourceInventoriedAs`